upgrade/Debian.go

```diff
@@ -0,0 +1,82 @@
+// Copyright © 2021 Alibaba Group Holding Ltd.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package upgrade
+
+import (
+	"fmt"
+
+	"github.com/alibaba/sealer/utils/ssh"
+)
+
+const (
+	aptUpdate      = `apt-get update`
```

Comment:
apt-get or yum don't have compatibility for other platfroms

Reply:
The Filesystem module will copy or the dependencies to all the nodes，so upgrade interface needn't care about install kubeadm and other bin files.